### ISSUE TITLE
ContextualVersionConflict starting SLEAP

### ISSUE THREAD

**Issue by johnmbarrett** [murthylab/sleap#367]
On trying to install SLEAP following the Windows instructions at https://sleap.ai/guides/installation.html, I was unable to start SLEAP.

Steps to reproduce:

1). `conda create -n sleap-test -c defaults -c sleap sleap=1.0 python=3.6 -y`
2). `conda activate sleap-test`
3). `sleap-label`

Expected behaviour: SLEAP launches.

Actual behaviour: SLEAP crashes with the following error:
```
Traceback (most recent call last):
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (scipy 1.5.0 (c:\users\lsps2\anaconda3\envs\sleap-test\lib\site-packages), Requirement.parse('scipy==1.4.1; python_version >= "3"'), {'tensorflow'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\Scripts\sleap-label-script.py", line 6, in <module>
    from pkg_resources import load_entry_point
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 3260, in <module>
    @_call_aside
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 3244, in _call_aside
    f(*args, **kwargs)
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 3273, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "C:\Users\LSPS2\Anaconda3\envs\sleap-test\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (scipy 1.5.0 (c:\users\lsps2\anaconda3\envs\sleap-test\lib\site-packages), Requirement.parse('scipy==1.4.1; python_version >= "3"'), {'tensorflow'})
```
Additional info:

Adding `scipy=1.4.1` to line 1 solves the issue. Also, possibly unrelated but whether I specify the scipy version or not I get an absolute tonne of ClobberErrors. However [this conda issue](https://github.com/conda/conda/issues/7038) suggests they aren't really a problem and it doesn't seem to affect the install (or running SLEAP if the correct scipy version is specified).

Here's my `conda info`:
```
     active environment : sleap-test
    active env location : C:\Users\LSPS2\Anaconda3\envs\sleap-test
            shell level : 2
       user config file : C:\Users\LSPS2\.condarc
 populated config files :
          conda version : 4.8.3
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       virtual packages : __cuda=10.1
       base environment : C:\Users\LSPS2\Anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\LSPS2\Anaconda3\pkgs
                          C:\Users\LSPS2\.conda\pkgs
                          C:\Users\LSPS2\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\LSPS2\Anaconda3\envs
                          C:\Users\LSPS2\.conda\envs
                          C:\Users\LSPS2\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.3 requests/2.22.0 CPython/3.7.4 Windows/10 Windows/10.0.18362
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by ntabris** [comment]
Hi, @johnmbarrett. Thanks for reporting this! Anaconda just got a new version of scipy yesterday which is causing this problem. I'm currently working on updating our conda package to fix this (and will let you know when it's up). In the mean time, you can fix the install by running `pip install scipy==1.4.1` (after activating the sleap conda environment).

**Comment by ntabris** [comment]
Fixed in SLEAP v1.0.4 (`scipy==1.4.1` is installed using `pip` inside the conda package we build).
